src/lib/ngsi/Scope.cpp

```diff
@@ -308,6 +316,21 @@ int Scope::fill
     return -1;
   }
 
+
+  //
+  // Georel 'near' only valid for a Point geometry
```

Comment:
Actually, near could work for not point geometries (it depend on the underlaying DB engine to solve the query). However, we could keep this restriction as it is by the moment and remove it once all the other things related with geo-queries are working.